pilot/pkg/security/authn/utils/utils.go

```diff
@@ -33,6 +33,16 @@ const (
 	PilotSvcAccName string = "istio-pilot-service-account"
 )
 
+var (
+	// SupportedCiphers for server side TLS configuration.
+	SupportedCiphers = []string{
```

Comment:
where does this come from? Envoy default? if so can we add a comment

Reply:
Its from OSWAP list of strong server side ciphers. All of them have been supported by envoy for a long time.